tests/components/controlBar-test.js

```diff
@@ -600,6 +816,95 @@ describe('ControlBar', function () {
     expect(buttons.length).toBeLessThan(5);
   });
 
+  it("hides the more options button when ooyala ad is playing", function() {
+    var mockController = {
+      state: {
+        isMobile: false,
+        volumeState: {
+          volume: 1
+        },
+        closedCaptionOptions: {},
+        videoQualityOptions: {
+          availableBitrates: null
+        },
+        isOoyalaAds: true
+      }
+    };
+
+    var oneButtonSkinConfig = Utils.clone(skinConfig);
+    oneButtonSkinConfig.buttons.desktopContent = [
+      {"name":"playPause", "location":"controlBar", "whenDoesNotFit":"keep", "minWidth":35 },
+      {"name":"playPause", "location":"controlBar", "whenDoesNotFit":"moveToMoreOptions", "minWidth":35 },
+      {"name":"playPause", "location":"controlBar", "whenDoesNotFit":"moveToMoreOptions", "minWidth":35 },
+      {"name":"playPause", "location":"controlBar", "whenDoesNotFit":"moveToMoreOptions", "minWidth":35 },
+      {"name":"playPause", "location":"controlBar", "whenDoesNotFit":"moveToMoreOptions", "minWidth":35 },
+      {"name":"moreOptions", "location":"controlBar", "whenDoesNotFit":"keep", "minWidth":35 }
+    ];
+
+    mockProps = {
+      isLiveStream: false,
+      controller: mockController,
+      skinConfig: oneButtonSkinConfig
+    };
+
+    DOM = TestUtils.renderIntoDocument(
+      <ControlBar {...mockProps} controlBarVisible={true}
+        componentWidth={100}
+        playerState={CONSTANTS.STATE.PLAYING}
+        isLiveStream={mockProps.isLiveStream} />
+    );
+
+    optionsButton = TestUtils.scryRenderedDOMComponentsWithClass(DOM, 'oo-more-options');
+    expect(optionsButton.length).toBe(0);
+    buttons = TestUtils.scryRenderedDOMComponentsWithClass(DOM, 'oo-play-pause');
+    expect(buttons.length).toBe(1);
```

Comment:
@aeng7 Because of the entry in oneButtonSkinConfig.buttons.desktopContent is to keep even when it doesn't fit the size
    oneButtonSkinConfig.buttons.desktopContent = [
      {"name":"playPause", "location":"controlBar", "whenDoesNotFit":"keep", "minWidth":35 },